README.md

```diff
@@ -1,42 +1,43 @@
 # terraform-rhcs-rosa-classic
 
-Terraform module which creates ROSA cluster
+A Terraform module which creates ROSA Classic clusters
 
 ## Notice
 
-This module is currently in its Alpha version and remains under active development; using it carries inherent risks that you should be aware of.
+This module is currently in Alpha and remains under active development. Using it carries inherent risks that you must be aware of.
 
-For the supported provider go to https://registry.terraform.io/providers/terraform-redhat/rhcs/latest
-For the supported rosa-sts module go to https://registry.terraform.io/modules/terraform-redhat/rosa-sts/aws/latest
+For information about the supported provider, go to https://registry.terraform.io/providers/terraform-redhat/rhcs/latest
+
+For information about the supported rosa-sts module, go to https://registry.terraform.io/modules/terraform-redhat/rosa-sts/aws/latest
```

Comment:
Please remove this section.
As once this PR is merge, this is not true anymore :-)